packages/rocketchat-cloud/server/functions/getWorkspaceAccessTokens.js

```diff
@@ -0,0 +1,43 @@
+import querystring from 'querystring';
+import { HTTP } from 'meteor/http';
+
+import { getRedirectUri } from './getRedirectUri';
+
+export function getWorkspaceAccessToken() {
+	if (!RocketChat.settings.get('Register_Server')) {
+		return '';
+	}
```

Comment:
We should probably add another check here to see if we have client_id otherwise its not actually registered

Reply:
Maybe call: retrieveRegistrationStatus and check the workspaceConnected?

Reply:
Great point 👍